_i18n/en/resources/user-guides/monero-wallet-cli.md

```diff
@@ -94,10 +94,20 @@ with that payment ID attached, so you can tell payments apart.
 It's suggested to use subaddresses (starting with `8`) instead of your main address (starting with `4`) to receive funds. Run:
 
 ```
-address new
+address new [<label text with white spaces allowed>]
 ```
 
-This will generate a subaddress, which you can share to receive payment on the account it's linked to. To view all generated addresses, run:
+This will generate a subaddress and its optional label, which addess you can share to receive payment on the account it's linked to.
+For example, 
+
+```
+
+address new github_donations
+
+```
+will generate a subaddress and its label 'github_donations'. 
+
+To view all generated addresses, run:
```

Comment:
Sorry, i realize my comment was misleading. This section should look like this:

````
For example,
(newline)

```
address new github_donations
```
(newline)
will generate a subaddress and its label 'github_donations'. 
````

Reply:
Yes, I think I know what you mean. So that the cited `address new github_donations` is surrounded by newlines in the same way as other code citations as I added. Please take a look now.